packages/lit-dev-cloudbuild-cleanup/src/cleanup.ts

```diff
@@ -0,0 +1,272 @@
+import {google} from 'googleapis';
+import {Octokit} from 'octokit';
+import {request} from 'gaxios';
```

Comment:
If Node 18 is available to us here, then we can use the new built in `fetch` API (same as the browser's API) so I think won't need gaxios.

https://nodejs.org/en/blog/announcements/v18-release-announce/

Reply:
Sounds reasonable. Didn't realize `gaxios` is a `googleapis` package itself.